### PR TITLE
 remove outermost parentheses in C

### PR DESCRIPTION
**Pull request by ag278**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Remove outermost parentheses

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
#955

### CODE REVIEW

**Comment by ag278** [comment]
@iamrajiv please have a review at my PR

**Review by iamrajiv — APPROVED** [review]

File: C/README.md [hunk 1]
```diff
@@ -70,10 +71,14 @@ _add list here_
 - [Random Search Monte Carlo version](search/random_search.c)
 
 
+
```
Comment (atarax665) [inline C/README.md:109]: Remove these unnecessary line breaks.

File: C/README.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ _add list here_
 ## Other
 - [Number_of_duplicates_using_Binary_search](other/Number_of_duplicates_using_Binary_search.c)
 - [Tower of Hanoi](other/Tower_Of_Hanoi.c)
+- [remove outermost parentheses](other/remove_outermost_parentheses.c)
```
Comment (atarax665) [inline C/README.md]:
```suggestion
- [Remove Outermost Parentheses](other/remove_outermost_parentheses.c)
```

**Comment by ag278** [comment]
@atarax665 @Abhijit2505 could you please review my PR

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by ag278** [comment]
@atarax665  i have made the required changes. Please verify it

File: C/README.md [hunk 3]
```diff
@@ -83,10 +84,21 @@ _add list here_
 - [Sum of Triplet](cp/sum_of_triplet.c)
 
 
+
+## Sorting
```
Comment (atarax665) [inline C/README.md]: Sorting heading already exists at line 41. Remove line 87 to 96.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by ag278** [comment]
@atarax665 i have made the required changes. Please verify it

**Comment by ag278** [comment]
@atarax665 @Abhijit2505 please review my pr

**Comment by ag278** [comment]
@atarax665 @Abhijit2505 @Kajol-Kumari @SKAUL05 please review my PR

**Pull request merged**